src/main/java/com/webcrawling/service/webCrawler/impl/WebCrawlerServiceImpl.java

```diff
@@ -71,14 +51,25 @@ public void  crawlWebsite(String startingUrl, int maxDepth, int maxDocuments) {
                         urlQueue.add(linkUrl);
                     }
                 } catch (IOException e) {
-                    //  logger.error("An error occurred while crawling URL: " + currentUrl, e);
+                      logger.error("An error occurred while crawling URL: " + currentUrl, e);
```

Comment:
While crawling web page we need to log exception into elastic search along with url and the reason of failure

Reply:
Got it